Cargo.toml

```diff
@@ -1,7 +1,14 @@
 [package]
 name = "vss-client"
 version = "0.1.0"
+license = "MIT OR Apache-2.0"
```

Comment:
We'll need to update our LICENSE files accordingly. You can follow-up `rust-lightning` as an example.